parquet/tests/arrow_reader/encryption_util.rs

```diff
@@ -17,7 +17,35 @@
 
 use arrow_array::cast::AsArray;
 use arrow_array::{types, RecordBatch};
+use parquet::arrow::arrow_reader::{
+    ArrowReaderMetadata, ArrowReaderOptions, ParquetRecordBatchReaderBuilder,
+};
+
+use parquet::arrow::ArrowWriter;
+use parquet::encryption::decrypt::FileDecryptionProperties;
+use parquet::encryption::encrypt::FileEncryptionProperties;
+use parquet::errors::Result;
 use parquet::file::metadata::ParquetMetaData;
+use parquet::file::properties::WriterProperties;
+use std::fs::File;
+
+/// Tests reading an encrypted file from the parquet-testing repository
+pub(crate) fn verify_encryption_test_file_read(
+    file: File,
+    decryption_properties: FileDecryptionProperties,
+) {
+    let options = ArrowReaderOptions::default()
+        .with_file_decryption_properties(decryption_properties.clone());
```

Comment:
No need to clone, `decryption_properties` aren't being used again in this function, they can be moved.
```suggestion
        .with_file_decryption_properties(decryption_properties);
```